velox/experimental/cudf/exec/DebugUtil.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/experimental/cudf/exec/DebugUtil.h"
+#include "velox/experimental/cudf/exec/Utilities.h"
+#include "velox/experimental/cudf/exec/VeloxCudfInterop.h"
+
+namespace facebook::velox::cudf_velox {
+
+std::string DebugUtil::toString(
+    const cudf::table_view& table,
+    vector_size_t from,
+    vector_size_t to) {
+  auto stream = cudfGlobalStreamPool().get_stream();
+  auto rowVector = with_arrow::toVeloxColumn(table, pool_.get(), "", stream);
+  stream.synchronize();
```

Comment:
You may want to pass a stream to this function and pass that to `toVeloxColumn`. That stream needs to be the same one that was used to make the `cudf::table` whose view is being passed here. Using a different stream from the global pool can give you wrong results.

Reply:
What amazing, I can print the correct result by current implementation. Maybe it would print wrong result in some cases. I will change it as you say.